Allow context sharing for headless contexts on MacOS (#1333)
rust-windowing/glutin: CHANGELOG.md

# Unreleased

- On Windows, fixed a panic for headless contexts because of active drag-and-drop (OleInitialize failed! Result was: `RPC_E_CHANGED_MODE`)

> - Context sharing of headless contexts is now supported on macOS.

Reviewer: ```suggestion
- On macOS, add support for sharing of headless contexts.
```

Reviewer 2: Thanks, that is clearer, applied.
